docs/how-to-guides/develop-a-tool/create-custom-llm-tool.md

```diff
@@ -0,0 +1,62 @@
+# Create Custom LLM Tool
+In this document, we will guide you through the process of developing a tool with `custom_llm` type which can connect to a customized large language model with prompt.
+
+Custom LLM is a large language model fine-tuned by yourself. If you find it has good performance and is useful, then you can follow this guidance to make it a tool so that it can be shared with other people to have more impact.
+
+## Prerequisites
+- Please ensure that your [Prompt flow for VS Code](https://marketplace.visualstudio.com/items?itemName=prompt-flow.prompt-flow) is updated to version 1.2.0 or later.
+
+## How to create a custom LLM tool
+Here we use [an existing tool package](https://github.com/microsoft/promptflow/tree/main/examples/tools/tool-package-quickstart/my_tool_package) as an example. If you want to create your own tool, please refer to [create and use tool package](create-and-use-tool-package.md).  
+
+1. Develop tool code like in [this example](https://github.com/microsoft/promptflow/blob/main/examples/tools/tool-package-quickstart/my_tool_package/tools/tool_with_custom_llm_type.py).
```

Comment:
Develop the tool code as in [this example]

Reply:
Changed, thanks.